src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -182,7 +182,7 @@ module.exports = withSentryConfig(moduleExports, SentryWebpackPluginOptions);
 
 <Alert level="warning" title="Serverless Environments">
 
-`@sentry/nextjs` does not support configurations with the `serverless` target. To use the SDK in serverless environments, switch to using the `experimental-serverless-trace` target, which is [recommended by the Next.js maintainers](https://github.com/vercel/next.js/issues/20487#issuecomment-753884085).
+`@sentry/nextjs` does not support configurations with the `serverless` target. To use the SDK in serverless environments, switch to using the `experimental-serverless-trace` target, which is [recommended by the Next.js maintainers](https://github.com/vercel/next.js/issues/20487#issuecomment-753884085) for apps using Next.js 10 or Next.js 11. (The Sentry SDK does not yet officially support Next.js 12.)
```

Comment:
This is redundant if we are adding the requirement on the getting started page.
```suggestion
`@sentry/nextjs` does not support configurations with the `serverless` target. To use the SDK in serverless environments, switch to using the `experimental-serverless-trace` target, which is [recommended by the Next.js maintainers](https://github.com/vercel/next.js/issues/20487#issuecomment-753884085) for apps using Next.js 10 or Next.js 11.
```

Reply:
Yes, but I thought it worth mentioning here as well because it's directly relevant to this issue (since Next 12 does away with `target` all together), and because "do this for these versions" begs the question of what the case is for the remaining version.

(Also, Lee specifically asked me to fix this for the Next 12 launch, so I figured being extra clear here wouldn't be a bad thing.)

Reply:
I still rather having each piece of info in one place, it makes it easier to maintain. But not strongly opinionated in this context, so your call.

Reply:
I also don't feel strongly, and as a general rule, you're right - the fewer spots to update the better. I can take it out.